study-builder/studies/pancan/study.conf

```diff
@@ -81,56 +81,70 @@
   },
 
   "governance": {
+    # This rule here is only used in the normal registration flow, not the add-child flow.
     "shouldCreateGovernedUserExpr": ${_pex.is_child_only}"""
       && (user.studies["cmi-pancan"].forms["CONSENT_PARENTAL"].hasInstance()
         || user.studies["cmi-pancan"].forms["CONSENT_ASSENT"].hasInstance())
     """,
+    # These rules apply to both normal registration and add-child flows, so check both.
     "ageOfMajorityRules": [
       # AoM for U.S.
       {
-        "condition": ${_pex.is_child_only}"&&"${_pex.is_country_us}"""
+        "condition": "("${_pex.has_prequal}"&&"${_pex.is_child_only}"&&"${_pex.is_country_us}"""
           && user.studies["cmi-pancan"].forms["PREQUAL"].questions["STATE"].answers.hasOption("AL")
+          ) || ("""${_pex.has_addchild}"&&"${_pex.addchild_is_country_us}"""
+          && user.studies["cmi-pancan"].forms["ADD_CHILD"].questions["CHILD_STATE"].answers.hasOption("AL"))
```

Comment:
nice that we can include ADD_CHILD to the AOM rules

Reply:
Assuming AOM triggers/events/actions like REACHED_AOM, GOVERNED_USER_REGISTERED , REVOKE_PROXIES should all fall in place for ADD_CHILD(s) or do we need to do any additional stuff for ADD_CHILD(s) !!

Reply:
The age-up events (`reached_aom`, `governed_user_registered`, etc) doesn't depend on `add_child` activity, so I don't see any issues there.